pkgs/development/ocaml-modules/stdlib-shims/default.nix

```diff
@@ -10,6 +10,9 @@ buildDunePackage rec {
   useDune2 = lib.versionAtLeast ocaml.version "4.08";
   minimumOCamlVersion = "4.02";
   doCheck = true;
+
+  passthru.tests = { inherit dose3; };
```

Comment:
Can you elaborate on the meaning/purpose/motivation of this line?

Reply:
When this package is upgraded, I expect it to test dose3 package as well.

Reply:
I think that `nixpkgs-review` already provides this functionality without requiring explicit “back-edges” in the dependency graph.

Reply:
I have copied it from another example on nixpkgs. What do you want me to do? Remove all these tests?
Just let me know and I'll remove it.

Reply:
I have removed all `passthru.tests` and clean-up `cudf`.
Enabling testing didn't go well, even partial.